src/test/regress/input/appendonly.source

```diff
@@ -822,9 +826,13 @@ explain select cmax, * from test_reporting;
 
 -- Replicated tables using different code pathes and we want to be sure than 
 -- all code pathes are covered by tests.
-drop table if exists test_reporting_r;
 create table test_reporting_r (id int) with (appendonly=true) distributed replicated;
 explain select xmin, * from test_reporting_r;
 explain select xmax, * from test_reporting_r;
 explain select cmin, * from test_reporting_r;
 explain select cmax, * from test_reporting_r;
+
+-- start_ignore
+drop table if exists test_reporting;
+drop table if exists test_reporting_r;
+-- end_ignore
```

Comment:
There is no need to add `if exists` and ignore at the end, as we can be sure that the tables exist.

Reply:
ignore is also not needed here